src/Controls/src/Core/Platform/Android/Extensions/ToolbarExtensions.cs

```diff
@@ -77,6 +77,9 @@ public static void UpdateTitleIcon(this AToolbar nativeToolbar, Toolbar toolbar)
 
 		public static void UpdateBackButton(this AToolbar nativeToolbar, Toolbar toolbar)
 		{
+			//Setting it to null to avoid the icon's blink
+			nativeToolbar.NavigationIcon = null;
```

Comment:
Interesting, do we know by chance where this is getting set upstream? I'm curious if we can fix this by not setting this upstream somewhere.

Reply:
I tried to look for it but couldn't find a better solution